index.qmd

```diff
@@ -110,6 +110,7 @@ Quarto files for working along [are available on GitHub](https://github.com/tidy
 
 ### English
 
+-   [August 2024](archive/2024-08-posit-conf/index.html) at posit::conf()
```

Comment:
For a moment, I saw the "plain HTML" kinda look (that is currently for the ConectaR slides, see below) if I don't specify `/index.html`. This doesn't click with my limited web understanding, but does the trick nonetheless.

![Screenshot 2024-09-06 at 3 47 05 PM](https://github.com/user-attachments/assets/a96fe559-e2b5-416d-bc8f-33262bc4ade4)

Reply:
I'm not sure what to make of this but for posit conf 2023, it does not link to `index.html` explicitly but it looks fine: https://workshops.tidymodels.org/archive/2023-09-posit-conf/.  I'm okay with you merging with or without. I suggest we just have a look at the deployed site to make sure everything is okay there 😆

Reply:
Just get the site a moment to build and then checked—we're in the clear!